src/index.ts

```diff
@@ -1,17 +1,59 @@
 import { Command } from 'commander';
 import { description, version } from '../package.json';
-import commands from './commands';
+
+function isObject(value: unknown): value is object {
+  return typeof value === 'object';
+}
+
+function isFunction(value: unknown): value is () => unknown {
+  return typeof value === 'function';
+}
```

Comment:
IMO you can inline these two.

Reply:
Inline like `if (typeof value === 'function')`? That sadly doesn't work, as I'm using this functions as type guards.

Reply:
After refactoring, this was now possible to inline.